style.css

```diff
@@ -197,7 +225,10 @@ hr {
 #works h2 {
   color: #9390ff;
   margin-bottom: 15px;
-  font-size: 2rem;
+  font-size: 2.3rem;
+  padding: 0 0 30px 0;
+  margin-right: 0;
+  max-width: 65%;
 }
 
 .superButton {
```

Comment:
- [ ] Your button color is not like the Figma design, please stick to the design as much as possible

![Screenshot_20220505_150242](https://user-images.githubusercontent.com/25227022/166953098-c3bd4eae-af35-49a8-a821-da8f6377240a.png) ![Screenshot_20220505_150217](https://user-images.githubusercontent.com/25227022/166953000-dc1508f1-4d9e-4a5b-9a5b-b6324549fb91.png)